[BE][ptd_fb_test][1/N] Enable testslide

fegin opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #126460

Testslide gives us more readable output, import time, etc.

Differential Revision: [D57410507](https://our.internmc.facebook.com/intern/diff/D57410507/)

fegin committed:
[BE][ptd_fb_test][1/N] Enable testslide

Testslide gives us more readable output, import time, etc.

Differential Revision: [D57410507](https://our.internmc.facebook.com/intern/diff/D57410507/)

[ghstack-poisoned]

wz337 reviewed (approved):
Stamped on internal already.

fegin committed:
Update on "[BE][ptd_fb_test][1/N] Enable testslide"


Testslide gives us more readable output, import time, etc.

Differential Revision: [D57410507](https://our.internmc.facebook.com/intern/diff/D57410507/)

[ghstack-poisoned]

fegin committed:
Update on "[BE][ptd_fb_test][1/N] Enable testslide"


Testslide gives us more readable output, import time, etc.

Differential Revision: [D57410507](https://our.internmc.facebook.com/intern/diff/D57410507/)

[ghstack-poisoned]

fegin commented:
There are some issues with ghexport, close this PR and create a new one.
